### ISSUE TITLE
Commit to branch not working on Github Desktop

### ISSUE THREAD

**Issue by PDJEY** [desktop/desktop#18531]
### The problem

For the past few days, despite trying to reeinstall the app, I simply can't commit to a repo, be it main or another branch. Already searched other reports but didn't find a fix.

### Release version

3.3.14(x64)

### Operating system

Windows 11

### Log files

[2024-04-29.desktop.production.log](https://github.com/desktop/desktop/files/15149506/2024-04-29.desktop.production.log)

### Screenshots

![image](https://github.com/desktop/desktop/assets/69373255/d3bec12a-61fc-4927-9281-883094490080)
![image](https://github.com/desktop/desktop/assets/69373255/b9a5a388-5498-4bb1-bd5a-ab968155ee91)

**Comment by steveward** [comment]
Thanks for the report @PDJEY. I'm seeing an exit code 1 in the log file:

```
2024-04-29T13:01:02.483Z - error: [ui] `git commit -F -` exited with an unexpected code: 1.
stdout:
On branch main
```

Unfortunately that isn't giving a lot of information about why committing is failing. Can you try committing directly from the command line to see if you encounter any issues there? Here's how to do that:

1. Go to `Repository` > `Open in <command_line>`
2. Run the command `git add -A`
3. Run the command `git commit -m "your commit message"

Let me know if that outputs any errors.

**Comment by PDJEY** [comment]
Doing that allows me to commit to then use the app to push to the repo. Does that mean I'm gonna have to stop using the app to commit?

**Comment by steveward** [comment]
That was mainly to see if the issue was present in both the command line and GitHub Desktop. Have you tried committing in GitHub Desktop again to see if that succeeds?

**Comment by PDJEY** [comment]
Yes the problem remains when trying to commit directly on GitHub desktop

**Comment by steveward** [comment]
Are you still encountering the same specific error? Have you also tried creating a new test repository to see if you encounter this issue there?

**Comment by PDJEY** [comment]
When trying a different repo the problem disappears :/

**Comment by steveward** [comment]
Closing as stale since this was isolated to a specific repository.
